### ISSUE TITLE
PCB Mounted Stabs Option

### ISSUE THREAD

**Issue by yanfali** [swill/kad#15]
Would be cool if you could specify PCB mounted stabilizers as an option to have a simpler cut out.

**Comment by jwhendy** [comment]
Yes, this was quite confusing to me as a first time user. The help page says:

> Cherry + Costar Stabilizer `{_s:1}`: A stabilizer cutout which supports; Cherry PCB Mount, Cherry Plate Mount and Costar stabilizers...

> Cherry Stabilizer `{_s:2}`: A stabilizer cutout which supports both Cherry PCB Mount and Cherry Plate Mount stabilizers...

But, there's no way this is correct, right? My Durock stabs hit the bottom of my plate and can't get up high enough such that the stems are the same height as the switches. I would need a cutout encompassing the full footprint of the stab (including the pcb clip + threaded insert), not just the stab body itself, at least for thicker plates.

Thanks for considering!
